egs/wsj/s5/steps/nnet3/chain/train_more.py

```diff
@@ -0,0 +1,590 @@
+#!/usr/bin/env python
+
+# Copyright 2016    Vijayaditya Peddinti.
+#           2016    Vimal Manohar
+# Apache 2.0.
+
+""" This script is based on steps/nnet3/chain/train.sh
```

Comment:
ok, I will do that, but I believe amount of changes to train.py is very low and we can still apply them to train.py!